src/Symfony/Component/DomCrawler/Field/ChoiceFormField.php

```diff
@@ -155,11 +155,11 @@ public function setValue($value)
     /**
      * Adds a choice to the current ones.
      *
-     * This method should only be used internally.
-     *
      * @param \DOMElement $node
      *
      * @throws \LogicException When choice provided is not multiple nor radio
+     *
+     * @internal Should not be used by user-land code.
```

Comment:
Comment doesn't add anything to the annotation, I'd suggest removing it, same below